strawberryfields/gbs/similarity.py

```diff
@@ -201,3 +203,42 @@ def event_to_sample(photon_number: int, max_count_per_mode: int, modes: int) ->
             available_modes.remove(j)
 
     return sample
+
+
+def feature_vector_sampling(
+    samples: list, event_photon_numbers: list, max_count_per_mode: int = 2
+) -> np.ndarray:
+    """Calculates feature vector with respect to input samples.
+
+    The feature vector is composed of event probabilities, with all events having a maximum
+    photon count in each mode of ``max_count_per_mode``. Events are then described by their total
+    photon number, and those chosen as part of the feature vector can be specified through
+    ``event_photon_numbers``. Probabilities are reconstructed by measuring the occurrence of
+    events in the input ``samples``.

```

Comment:
Is it worth adding an equation here?

Reply:
Might be easier to parse than sentences :thinking:

Reply:
I added equations 😁